Build fails with IRkernel

The build using `make images` errors with the following message at the end.

```
The downloaded source packages are in
	‘/tmp/RtmpOsVzqo/downloaded_packages’
'/usr/lib/R/bin/R' --vanilla CMD INSTALL  \
  '/tmp/RtmpOsVzqo/devtools7607407a6/IRkernel-IRkernel-676efc7'  \
  --library='/home/jovyan/.R' --install-tests 

* installing *source* package ‘IRkernel’ ...
** R
** inst
** preparing package for lazy loading
Error in loadNamespace(i, c(lib.loc, .libPaths()), versionCheck = vI[[i]]) : 
  there is no package called ‘repr’
ERROR: lazy loading failed for package ‘IRkernel’
* removing ‘/home/jovyan/.R/IRkernel’
Error: Command failed (1)
Execution halted
INFO[0053] The command [/bin/sh -c echo "library(devtools); install_github('armstrtw/rzmq'); install_github('IRkernel/IRdisplay'); install_github('IRkernel/IRkernel'); IRkernel::installspec()" | R --no-save] returned a non-zero code: 1 
make: *** [demo-image] Error 1
```

--- a/Dockerfile
+++ b/Dockerfile
@@ -56,10 +56,10 @@ RUN echo "install.packages(c('ggplot2', 'XML', 'plyr', 'randomForest', 'Hmisc', 
 RUN echo "install.packages(c('RCurl', 'devtools', 'dplyr'))" | R --no-save
 RUN echo "install.packages(c('httr', 'knitr', 'packrat'))" | R --no-save
 RUN echo "install.packages(c('rmarkdown', 'rvtest', 'testit', 'testthat', 'tidyr', 'shiny'))" | R --no-save
-RUN echo "library(devtools); install_github('armstrtw/rzmq'); install_github('IRkernel/IRdisplay'); install_github('IRkernel/IRkernel'); IRkernel::installspec()" | R --no-save
+RUN echo "library(devtools); install_github('armstrtw/rzmq'); install_github('IRkernel/IRdisplay');  install_github('IRkernel/repr'); install_github('IRkernel/IRkernel'); IRkernel::installspec()" | R --no-save
 RUN echo "library(devtools); install_github('hadley/lineprof')" | R --no-save
 RUN echo "library(devtools); install_github('rstudio/rticles')" | R --no-save
-RUN echo "library(devtools); install_github('jimhester/covr')" | R --no-save
+RUN echo "library(devtools); install_github('jimhester/robustr'); install_github('jimhester/covr')" | R --no-save
 
 RUN echo "install.packages(c('base64enc', 'Cairo', 'codetools', 'data.table', 'gridExtra', 'gtable', 'hexbin', 'jpeg', 'Lahman', 'lattice'))" | R --no-save
 RUN echo "install.packages(c('MASS', 'PKI', 'png', 'microbenchmark', 'mgcv', 'mapproj', 'maps', 'maptools', 'mgcv', 'multcomp', 'nlme'))" | R --no-save

